epub_test.go

```diff
@@ -19,7 +19,7 @@ import (
 
 const (
 	// Set this to false to not delete the generated test EPUB file
-	doCleanup             = true
+	doCleanup             = false
```

Comment:
We should probably leave this set to `true` to avoid leaving the generated test file

Reply:
I agree, this is a mistake